test-it/ViaTests/TestVI_VariantDeleteAttributes.via

```diff
@@ -153,7 +153,7 @@ define (Application%3A%3ATestVI_VariantDeleteAttributes%2Egviweb dv(.VirtualInst
         ce(dv(.Boolean true)c146)
         ce(dv(.ErrorCluster (false 0 '' ))c147)
         e(.Boolean local148)
-        ce(dv(.Boolean true)c149)
+        ce(dv(.Boolean false)c149)
```

Comment:
This is related to the Case 2 I described above. This comes from a cross-target test that worked on both desktop and vireo previously, but incorrectly. It was returning `found = false` when no attributes were deleted (From a variant all of whose attributes were already deleted. This was trying to delete them a second time). The test passed in desktop because of bug 2, and it passed in Vireo because of the bug 1, as described in the overview of this PR.